client/api/python/heketi/heketi.py

```diff
@@ -54,3 +56,535 @@ def hello(self):
         r = requests.get(self.host + uri, headers=headers)
         return r.status_code == requests.codes.ok
 
```

Comment:
One overall comment.  There seem to be objects for each endpoint type.  This would require the client to keep creating objects for each endpoint.  Instead, all those endpoints should be functions under the _Client_ class.  That way it would enable the client to create a single initialized object which has access to all api functions.  Also it would enable the member functions to access self._set_toket_in_header() and other utility functions (like pending for the request to complete).